README.md

```diff
@@ -1,2 +1,50 @@
-# Portfolio-Project
-This is a recurring updated Portfolio of mine.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: Mobile first
+
+> Understand how to parse a Figma design to create a UI.
+>Flexbox to replicate designs.
+>Build a personal portfolio site.
+>Use images and backgrounds to enhance the look of the website.
+
+
+## Built With
+
+- HTML/SS
+- Figma
+- VSCode
+
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your license is broken :cry:  Please don't forget to add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/) . :smile: